dev/customWidgets/hitsBel.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/default */
+import instantsearch from '../../index.js';
+import bel from 'bel';
+
+function render({hits, widgetParams: {containerNode}}) {
+  const content = bel`<div>
+    ${hits.map(hit => {
+      const title = bel`<h4></h4>`;
+      title.innerHTML = hit._highlightResult.name.value;
+      const description = bel`<p></p>`;
+      description.innerHTML = hit._highlightResult.description.value;
+
+      return bel`
+        <div class="ais-hits--item">
+          <div class="hit" id="hit-bel-${hit.objectID}">
+            <div class="media">
+              <a class="pull-left" href="${hit.url}">
+                <img class="media-object" src="${hit.image}">
+              </a>
+              <div class="media-body">
+                <h3 class="pull-right text-right text-info">$${hit.price}</h3>
+                <h4>${title}</h4>
+                <p>${description}</p>
+                ${hit.free_shipping ? bel`<span class="badge pull-right">Free Shipping</span>` : ''}
```

Comment:
You do need to use it again 😄, otherwise it's just a template string, so just text